bigbluebutton-html5/imports/ui/components/settings-loader/component.tsx

```
@@ -59,36 +48,36 @@ const SettingsLoader: React.FC<SettingsLoaderProps> = (props) => {
+        },
+        signal: controller.signal,
+        timeout: connectionTimeout / 3,
+        retries: 3,
+        retryDelay: 1000,
+      },
+    )
+      .then((response) => response.json())
+      .then((data) => {
+        clearTimeout(timeoutRef.current);
+
+        const { clientSettings, ...staticData } = data?.meeting[0];
```

Biome (2.1.2)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)